### PR TITLE
Prohibit PATCH on LDPCv

### PR DESCRIPTION
**Pull request by awoods**
Resolves: https://github.com/fcrepo/fcrepo-specification/issues/243

### CODE REVIEW

**Comment by birkland** [comment]
What is the rationale or underlying principle for prohibiting PATCH on LDPCv?

**Comment by awoods** [comment]
@birkland : I believe the idea is that TimeMaps serve the purpose of containing and listing mementos, period.

**Comment by birkland** [comment]
I think that's too restrictive to set in stone.  Implementations have all the tools they need to lock it down anyway.  For example, is it not allowable _by definition_ to want to call the LDPCv an `ore:Aggregation' (or some other concept that makes sense, according to some ontology)?  Likewise, is it inherently not allowable to do that retroactively via PATCH?

It may be best to just not mention anything (i.e. remove the sections on PATCH).

**Comment by awoods** [comment]
@birkland : I would be fine with dropping the PATCH mentions for LDPCv-s.
@fcrepo/editors : thoughts?

**Comment by escowles** [comment]
Since LDPCv is still a LDPC, it seems good to preserve as much functionality as possible — so I'm fine with leaving those sections as-is.

**Comment by zimeon** [comment]
I'm also OK with leaving PATCH description as-is. Propose closing this PR and #243

**Comment by awoods** [comment]
@birkland @escowles @zimeon : It seems that we are discussing three options as it relates to PATCH on LDPCv-s:
1. Prohibit PATCH (this PR)
2. Be silent on PATCH ( @birkland suggestion )
3. Leave the current text ( @escowles suggestion )

I am inclined towards option-2 as it is not clear to me under what circumstances the user would see user-triples on a TimeMap, given the strict [specification of the contents of the TimeMap](https://tools.ietf.org/html/rfc7089#page-36), but could live with option-3 if that is the consensus view.

**Comment by escowles** [comment]
@awoods We define LDPCv as "A version container: an LDPC that is simultaneously a Memento TimeMap." — since it is an LDPC, then I would expect all of the normal LPDC functionality to work normally.

Memento [section 5](https://tools.ietf.org/html/rfc7089#page-36) says the TimeMap:
* MUST list the URI-R/Original Resource, and the URI-M for each Memento, and unambiguously identify the types of all resources.
* MUST be serializable in the Memento link-value format, but MAY be serializable in other formats.
* SHOULD also include the URI-G/TimeGate and URI-T/TimeMap.

So, I think an RDF description of the LDPRv, LDPCv, and all LDPRms in various RDF formats would fulfill those requirements, and including any other triples that the user had supplied doesn't seem like it would run afoul of the requirements either.

In short, I think the LDPCv can just be a normal container, with some added Memento functionality.  Is there any requirement from LDP or Memento that we suppress any functionality?  Is there any use case for doing so?

**Comment by awoods** [comment]
@escowles : I was thinking mostly about the link-format requirement for TimeMaps:

> The entity-body of a response from a TimeMap MAY be serialized in
   various ways, but the link-value format serialization described here
   MUST be supported.  In this serialization, the entity-body MUST be
   formatted in the same way as the value of an HTTP "Link" header, and
   hence MUST comply to the "link-value" construction rule of Section 5.
   The Link header field of [RFC5988], and the media type of the entity-
   body MUST be "application/link-format" as introduced in [RFC6690].

..and the implication that requirement brings to the content of the TimeMap. However, I could get behind either option-2 or option-3.

**Comment by zimeon** [comment]
@awoods I think it is reasonable that the link format of the TimeMap have only the Memento parts, but that RDF forms might include other triples. I lean slightly towards option-3 (I fear silence leaves obvious questions unanswered) but could also live with option-2.

**Comment by escowles** [comment]
I agree with @zimeon that the link-format of the TimeMap should just have the Memento parts, but the RDF serializations could have other triples.  That's why I think we should keep the current text, since it calls out the possibility of allowing properties on the TimeMap, and identifies the pitfall of allowing modification of the containment triples, which implementations need to avoid.

**Comment by barmintor** [comment]
Should this instead be: "an implementation MUST NOT allow PATCH to modify a LDPCv's containment triples"? It's ambiguous in the LDP spec, but especially problematic in this context. The issue if user triples I prefer to be silent on.

**Comment by awoods** [comment]
Given the diversity of options and opinions here, I have added the topic to discuss during the next API Spec meeting: https://wiki.duraspace.org/display/FF/2017-10-25+-+API+Spec+Meeting

**Comment by barmintor** [comment]
It's been pointed out to me that my suggestion is already in the spec for LDPCv's at 4.3.4 in the 20 Oct draft.

File: index.html [hunk 1]
```diff
@@ -766,14 +770,6 @@
           </p>
         </section>
 
-        <section id="ldpcvpatch">
```
Comment (zimeon) [inline index.html:769]: The anchor `#ldpcvpatch` is referenced so that needs to be addressed to allow removal. Otherwise I'm :+1: on the change.

Comment (awoods) [reply index.html:769]: Updated with: https://github.com/fcrepo/fcrepo-specification/pull/244/commits/cc00a85643e6257d418789c2d9eefdf50361381e

**Review by zimeon — DISMISSED** [review]

**Review by escowles — DISMISSED** [review]

**Review by escowles — APPROVED** [review]

**Review by zimeon — APPROVED** [review]

**Comment by zimeon** [comment]
Majority approval, no dissenting, was 3 days old before syntax fix -- will merge
